model/flow/event.go

```diff
@@ -79,6 +79,16 @@ func wrapEventID(e Event) eventIDWrapper {
 	}
 }
 
+// byteSize returns the number of bytes needed to store the wrapped version of the event.
+// returned int is an approximate measure, ignoring the number of bytes needed as headers.
+func (e Event) byteSize() int {
+	return IdentifierLen + // txID
+		4 + // Index
+		len(e.Type) + // Type
+		4 + // TransactionIndex
+		len(e.Payload) // Payload
+}
```

Comment:
this is an approximate size, I was trying to avoid encoding to capture the size, ideally we could do something smarter based on how we encode and store events.

Reply:
we actually do collect the size when emitting the events, maybe we could consolidate and capture the size here as a field.

Reply:
+1 wrt janez's comment